app/views/assignments/_timed_due_info.html.erb

```diff
@@ -0,0 +1,40 @@
+<% due_date = @assignment.section_due_date(@current_user&.section)
+   start_time = @assignment.section_start_time(@current_user&.section) %>
+<% duration = @grouping.nil? ?
+                @assignment.assignment_properties.adjusted_duration(due_date, [Time.current, start_time].max) :
+                @grouping.adjusted_duration %>
+<h3><%= I18n.t('activerecord.models.timed_assignment.one') %></h3>
+<p>
+  <strong><%= I18n.t('assignments.timed.start_time_instructions') %></strong>
+  <%= @assignment.section_start_time(@current_user&.section).strftime('%Y-%m-%d %l:%M %p') %>
```

Comment:
In general in the views it would be preferable to use `I18n.l` rather than `strftime` I think.